docs/EN_US/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml

```diff
@@ -271,23 +271,30 @@ ECLCC_PATH
               </tgroup>
             </informaltable></para>
         </sect3>
+      </sect2>
+
+      <sect2 id="CT-CLI-GIT-Support" role="brk">
+        <title>Git Support</title>
+
+        <para>HPCC Systems integrates native support for leveraging
+        Git.</para>
 
-        <sect3 id="CT-CLI-GIT-Support">
-          <title>Git Support</title>
+        <para>The <emphasis>--main</emphasis> option has been extended to
+        support compiling a query directly from a Git repository. When
+        invoked, It retrieves the ECL code specified from the Git repository,
+        compiles the code, and runs the query. Only the checkout will be done
+        on the remote ECLCCServer rather than on the client machine.</para>
```

Comment:
Not a complete sentence: Only the checkout will be done on the remote ECLCCServer rather than on the client machine. 
How about:
The checkout is done on the remote ECLCCServer rather than on the client machine.